src/main/java/com/minelittlepony/mixin/MixinEntityLivingBase.java

```diff
@@ -0,0 +1,39 @@
+package com.minelittlepony.mixin;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+
+import com.minelittlepony.ducks.IPonyAnimationHolder;
+
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.world.World;
+
+@Mixin(EntityLivingBase.class)
+public abstract class MixinEntityLivingBase extends Entity implements IPonyAnimationHolder {
+    @Shadow
+    public float moveStrafing;
+
+    private MixinEntityLivingBase(World worldIn) {
+        super(worldIn);
+    }
+
+    // No other place to save this stuff? :'c
+    // Add any animations you want
+    // This could also go into Pony, but I'm unsure if that's a good place for it (@Immutable).
+    private float strafeRollAmount = 0;
```

Comment:
You can put this in the renderer. Fly/elytra angle calculations are also done in there.

Reply:
Not neccessarily. There's a value that needs to be saved between render calls for the entity.